provision-vm-service/provision-vm.asl

```diff
@@ -31,7 +32,6 @@
       },
       "Parameters": {
         "VCENTER_HOST.$": "$.vcenter_host",
-        "VERIFY_SSL.$": "$.dialog_verify_ssl",
```

Comment:
Turns out this wasn't used, and it was confusing because it sounded like we should be verifying SSL for vcenter itself, even though `insecure: true` is being sent anyway.  Even if this was meant to be verifying the SSL of vcenter itself, `dialog_verify_ssl` is meant for the ManageIQ API, so I would have expected a separate `dialog_verify_vcenter_ssl`.